source/seedsource_core/django/seedsource/tasks/generate_scores.py

```diff
@@ -44,12 +44,14 @@ def __init__(self):
 
     def load_variable_data(self, variable, region, year, model=None):
         if variable == "LAT":
-            with Dataset(
-                os.path.join(
-                    NC_SERVICE_DIR, "regions", region, "{}_dem.nc".format(region)
+            service = Service.objects.get(name=f"{region}_dem")
+            with Dataset(str(Path(NC_SERVICE_DIR) / service.data_path)) as ds:
+                variable = service.variable_set.all()[:1].get()
+                return create_latitude_data(
+                    SpatialCoordinateVariables.from_dataset(
+                        ds, x_name=variable.x_dimension, y_name=variable.y_dimension
+                    )
```

Comment:
This change is because the new data, processed with gdal utilities, has different coordinate variable names (e.g., `longitude` instead of `lon`).